pkg/alerts/muting_rules.go

```diff
@@ -0,0 +1,175 @@
+package alerts
+
+// MutingRule represents the alert suppression mechanism in the Alerts API.
+type MutingRule struct {
+	ID          int                      `json:"id,string,omitempty"`
+	AccountID   int                      `json:"accountId,string,omitempty"`
```

Comment:
Looks like `accountId ` is returned as type `Int` in the NerdGraph response. Do we want to unmarshal this to a string per the current JSON tag? Seems like we would want to unmarshal it to an `int`.

Reply:
Good catch.  I'm surprised it worked at all.